acceptance/tests/windows/QA-563_windows_exit_mcollective.rb

```diff
@@ -52,17 +52,13 @@
 step 'Shutdown MCollective service on Windows agent'
 #Shutdown MCollective Service on Windows agent and make sure it successfully exits
 agents.each do |agent|
-  on agent, 'net stop mcollective' do |result|
-    assert_match(/The Marionette Collective Server service was stopped successfully/, result.stdout, "Failed to stop MCollective service")
-  end
+  on agent, 'net stop mcollective'
```

Comment:
@glennsarti has pointed out that Beaker defaults to only accepting exit code of `0` per https://github.com/puppetlabs/beaker/blob/dbc5be2d107f58373dbf4a9942a925a1a8313e7b/docs/concepts/shared_options_for_executing_beaker_commands.md#acceptable_exit_codes

I think it might still be a bit more self-documenting to explicitly specify them in the test, so that it's clear an actual assertion happens.